transports/jetty/src/main/java/org/mule/transport/servlet/jetty/JettyMessageProcessTemplate.java

```diff
@@ -36,6 +37,7 @@
     private final HttpServletResponse servletResponse;
     private final MuleContext muleContext;
     private boolean failureResponseSentToClient;
+    private Map<String, String> extraHeaders = new HashMap<String, String>();
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace the type specification in this constructor call with the diamond operator ("<>"). ([squid:S2293](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS2293))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7625&issues=AWiOv--GotIXnmsU1tFx&open=AWiOv--GotIXnmsU1tFx)

[ISSUE_KEY]: AWiOv--GotIXnmsU1tFx